server/pension/tests.py

```diff
@@ -1,3 +1,7 @@
 from django.test import TestCase
 
-# Create your tests here.
+class TestPension(TestCase):
```

Comment:
@ranmoshe 

Since we're testing the __API__ over here please move this file to be under the `/api` app and rename the class, maybe something like `TestOpenPensionAPI` ?

thanks.